packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -14,6 +14,7 @@ import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
 // Interfaces
 import { IResourceMetering } from "interfaces/L1/IResourceMetering.sol";
 import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
+import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
```

Comment:
Need to add tests for:
- `superchainConfig()`
- `guardian()`
- `paused(address)`